sites/all/modules/reol_widget/src/Controller/SearchController.php

```diff
@@ -211,4 +216,117 @@ private function getUrl(array $query = [], array $unset = []) {
     return url(current_path(), ['absolute' => TRUE, 'query' => $query]);
   }
 
+  /**
+   * Get valid sort key from query parameters.
+   *
+   * @param array $parameters
+   *   The query parameters.
+   * @param string $default
+   *   The default sort key.
+   *
+   * @return null|string
+   *   The sort key if any and valid.
+   */
+  private function getSort(array $parameters, $default = 'date_created_descending') {
```

Comment:
The `TingSearchCommonFields` in SAL have the fields I thinks is available. Possible check with @sgl